### PR TITLE
Algo Data import

### LINKED ISSUE #83: Add Algo required data files to repository
Make sure data files needed to run algorithms are in the backend repository for database initialization.

### PR DESCRIPTION
**Pull request by ekanemeinstein**
# Description
Imported Algo data files to our repository for database initialization.
Fixes #83 
## Type of change
Please delete options that are not relevant.
- [x] N/A
# How Has This Been Tested?
N/A
# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have linted my code and fixed all issues

### CODE REVIEW

**Review by zoegoodwinsutton — APPROVED** [review]
LGTM

File: database/init_data/algo_data/previous_enrolment.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        "3": 92,
+        "4": 55,
+        "5": 50,
+        "6": 8,
+        "7": 2
+    },
+    "2021": {
+        "1": 113,
+        "2": 90,
+        "2T": 15,
```
Comment (owenjaques) [inline database/init_data/algo_data/previous_enrolment.json:75]: Out of curiosity, what is 2T?

**Review by owenjaques — APPROVED** [review]
Looks good :) A little confusing what each number means but I understand this is the format they want

File: database/init_data/algo_data/previous_enrolment.json (hunk 1 above)
Comment (owenjaques) [reply database/init_data/algo_data/previous_enrolment.json:75]: Nvm figured it out (second year transfers)

**Pull request merged**